src/lib/stark/ledger/createSignedTransfer.js

```diff
@@ -49,6 +51,9 @@ module.exports = async (
     transferTokenAddress = null
   }
 
+  console.log({transferTokenAddress, transferQuantization})
+  await eth.starkProvideQuantum(transferTokenAddress, transferQuantization)
```

Comment:
what level of testing has this had?

Reply:
@plutoegg I am keeping some console logging for this change as this is for debugging. console logging will be removed when we have tested.